crates/ruff_python_formatter/src/statement/suite.rs

```diff
@@ -734,7 +741,16 @@ pub(crate) struct DocstringStmt<'a> {
 
 impl<'a> DocstringStmt<'a> {
     /// Checks if the statement is a simple string that can be formatted as a docstring
-    fn try_from_statement(stmt: &'a Stmt, suite_kind: SuiteKind) -> Option<DocstringStmt<'a>> {
+    fn try_from_statement(
+        stmt: &'a Stmt,
+        suite_kind: SuiteKind,
+        source_type: PySourceType,
+    ) -> Option<DocstringStmt<'a>> {
+        // Notebooks don't have a concept of notebooks, never recognise them as docstrings.
```

Comment:
I believe you meant to include "module" somewhere in here?

Reply:
And here I'm pretending that jetlag isn't affecting me